grails-app/domain/org/pih/warehouse/product/ProductSupplier.groovy

```diff
@@ -61,6 +61,12 @@ class ProductSupplier implements Serializable, Comparable<ProductSupplier> {
     // Additional comments
     String comments
 
+    // Documents Availability
+    Boolean coaAvailable = Boolean.FALSE
+    Boolean isoAvailable = Boolean.FALSE
```

Comment:
When we talked about this either (during planning poker?) I said that I wanted this to be done using the Attribute domain in manner similar to ProductAttribute. It doesn't look like that was recorded in the ticket so assign this to me and I'll do it myself.

Reply:
I forgot about this. I think could fix that, but I need a broader understanding of how you see it.